.travis.yml

```diff
@@ -45,6 +45,16 @@ jobs:
         - docker build -t openhabian/dev-openhabian -f Dockerfile.openhabian-aarch64-native .
         - docker run -i openhabian/dev-openhabian bash -c '/usr/local/bin/bats -r -f "dev-." .'
         - docker run -i openhabian/dev-openhabian bash -c '/usr/local/bin/bats -r -f "unit-." .'
+    - env: aarch64
+      arch: arm64
+      script:
+        # 64bit Azul Java on 64bit ARMv8 (RPi3,4) on 64bit OS on Travis Arm64 HW
+        - export HW=rpi4
+        - sed -i 's/java_arch=64-bit$/java_arch=AdoptOpenJDK/' build-image/openhabian.conf
```

Comment:
I don't think it's a good idea to reuse java_arch
We have  many places in the code where we replace the contents, including build.bash, these we all would have to validate

Reply:
Its easiest to use that one because it allows system-wide tracking of the Java version. I could rename it to be more applicable.

Reply:
I'm undecided for now.
I see the point of having a single parameter for that.
*If* you ensure it's working in *all* places I'd be fine with that, but there you must pay
big attention. Include build.bash, test.bash and all the rest where it might be in use.

Yes you would need to rename it (simply "java" I'd think).
You would also have to rename "32-bit" and "64-bit" to Zulu32 and Zulu64 or similar.

Reply:
I'll work on that, then.

Reply:
It does not appear to be in use anywhere else throughout the code, other than in `menu.bash` and `.travis.yml` which have both been updated.

Reply:
Careful:
build-image/openhabian.*
functions/config.bash,system.bash

Reply:
Yep I got those.